phpcs.xml

```diff
@@ -39,9 +39,32 @@
 	<arg name="extensions" value="php"/>
 	<file>.</file>
 
+	<!--
+		Normally the exclude patterns for directories should end in /* however there is a bug
+		in the current version of phpcs (see https://github.com/squizlabs/PHP_CodeSniffer/issues/1731)
+		which breaks this when a single file name is passed to be checked (for example with TravisCI).
+
+		A fix has been committed and will be included in phpcs version 3.2, however this is no release
```

Comment:
PHPCS 3.2 was released in December 2017, and the latest release is 3.2.3, with 3.3.0 not too far away. It would be reasonable to make `^3.2` the minimum version of PHPCS required for devs to contribute to GlotPress.

Reply:
This should have automatically taken care of itself, we just have to remove the extra lines now as they were only needed before 3.2 was released.